### DISCUSSION TITLE
Chat component from northstar is missing

### DISCUSSION THREAD

**Discussion by jackblackCH** [microsoft/fluentui#25832] [Q&A] [answered]
I am migrating to fluent-ui v9 but I can't find the chat component in fluentUi v9, or any migration readme about.

- How should the existing Chat component be migrated? 
- Something about missing components should be added to the migration sections.

**Comment by JustSlone** [comment, marked as answer]
Regarding the migration path today I'll defer to @jurokapsiar and @YuanboXue-Amber 

Regarding the planning, good call, when we first did planning for v9 Chat was a prototype/example component (as I recall). Since it's a full-fledged component now we should make a call on whether we'll implement it in v9 or what the migration path is.
